acceptance/tests/wan-federation/wan_federation_test.go

```diff
@@ -190,11 +209,259 @@ func TestWANFederation(t *testing.T) {
+			logger.Log(t, "Verifying federation was successful")
+			helpers.VerifyFederation(t, primaryClient, secondaryClient, releaseName, c.secure)
+
+			// Create a ProxyDefaults resource to configure services to use the mesh
+			// gateways.
+			logger.Log(t, "Creating proxy-defaults config")
+			kustomizeDir := "../fixtures/bases/mesh-gateway"
+			k8s.KubectlApplyK(t, secondaryContext.KubectlOptions(t), kustomizeDir)
+			helpers.Cleanup(t, cfg.NoCleanupOnFailure, cfg.NoCleanup, func() {
+				k8s.KubectlDeleteK(t, secondaryContext.KubectlOptions(t), kustomizeDir)
+			})
+
+			primaryHelper := connhelper.ConnectHelper{
+				Secure:          c.secure,
+				ReleaseName:     releaseName,
+				Ctx:             primaryContext,
+				UseAppNamespace: false,
+				Cfg:             cfg,
+				ConsulClient:    primaryClient,
+			}
+			secondaryHelper := connhelper.ConnectHelper{
+				Secure:          c.secure,
+				ReleaseName:     releaseName,
+				Ctx:             secondaryContext,
+				UseAppNamespace: false,
+				Cfg:             cfg,
+				ConsulClient:    secondaryClient,
+			}
+
+			// Create Namespaces
+			primaryNamespaceOpts := primaryHelper.Ctx.KubectlOptionsForNamespace(primaryNamespace)
+			primaryHelper.SetupNamespace(t, primaryNamespaceOpts.Namespace)
+			primarySecondaryNamepsaceOpts := primaryHelper.Ctx.KubectlOptionsForNamespace(secondaryNamespace)
+			primaryHelper.SetupNamespace(t, primarySecondaryNamepsaceOpts.Namespace)
+			secondaryNamespaceOtps := secondaryHelper.Ctx.KubectlOptionsForNamespace(primaryNamespace)
+			secondaryHelper.SetupNamespace(t, secondaryNamespaceOtps.Namespace)
+
+			// Create a static-server in dc2 to respond with its own name for checking failover.
+			logger.Log(t, "Creating static-server in dc2")
+			k8s.DeployKustomize(t, secondaryNamespaceOtps, cfg.NoCleanupOnFailure, cfg.NoCleanup, cfg.DebugDirectory, "../fixtures/cases/wan-federation/dc2-static-server")
+
+			// Spin up a server on dc1 which will be the primary upstream for our client
+			logger.Log(t, "Creating static-server in dc1")
+			k8s.DeployKustomize(t, primaryNamespaceOpts, cfg.NoCleanupOnFailure, cfg.NoCleanup, cfg.DebugDirectory, "../fixtures/cases/wan-federation/dc1-static-server")
+			logger.Log(t, "Creating static-client in dc1")
+			k8s.DeployKustomize(t, primaryNamespaceOpts, cfg.NoCleanupOnFailure, cfg.NoCleanup, cfg.DebugDirectory, "../fixtures/cases/wan-federation/static-client")
+
+			// Spin up a second server on dc1 in a separate namespace
+			logger.Logf(t, "Creating server on dc1 in namespace %s", primarySecondaryNamepsaceOpts.Namespace)
+			k8s.DeployKustomize(t, primarySecondaryNamepsaceOpts, cfg.NoCleanupOnFailure, cfg.NoCleanup, cfg.DebugDirectory, "../fixtures/cases/wan-federation/dc1-ns2-static-server")
+
+			// There is currently an issue that requires the intentions and resolvers to be created after
+			// the static-server/clients when using namespaces. When created before, Consul gives a "namespace does not exist"
+			// error
```

Comment:
~ Do we have a Jira/GH issue to reference for when this is fixed? NBD if not but might be useful to future-us

Reply:
No we don't, I don't know if we create github issues? and I don't want to link a jira issue in a public repo. I'll create a jira issue for it.